.github/actions/build/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Container image that runs your code
+FROM adoptopenjdk/openjdk15
```

Comment:
Question for @abekornelis @dsh33782 @jyganci - what version of Java do you want to use to build the package and test. As you can see, I am currently using AdoptOpenJDK v15. 
Options are 8 (LTS), 11 (LTS) or 16 (Latest version). I would suggest the latest LTS version (LTS=Long term support)

Reply:
I vote for Java 8. I use Oracle Java 1.8.0_201 which uses the older license and agree with Anthony that the current Oracle license should be avoided. Since the Java 8 update 8u221 on April 16, 2019, the Oracle JDK license changed to a more restrictive license. See the Oracle Java download page.

Hugh's comment about using "javac -source 8 -target 8 -encoding UTF-8" makes me wonder whether it should be incorporated into BLDJAR.BAT and corresponding bash script bldjar. This would allow a developer to use any free version of the JDK (Oracle JDK 8 prior to 8u221, any OpenJDK 8+ version) to build z390 for any JRE 8+ that a user might have.

Reply:
Hugh is entirely right. I;m in favor of following his advice.